### ISSUE TITLE
Cover paragraph bundles with section background colors

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#1654]
## Problem

Section background color choices can have cascading affect on nested paragraph bundles resulting in a11y concerns.

## Proposed Solution

Check and address these cascading and white background issues appropriately similar to uiowa_boostrap but with a lighter footprint.

Recently, attention was given to articles, events, featured content, people. Apply the same solutions while accounting for complicated combinations like a change to text/blockquote backgrounds in addition to the parent section background.

- [x] Section (Default)
  - [ ] Text (has background color options, can contain blockquotes)
  - [ ] Blockquote
  - [ ] Webform (has background color options)
  - [ ] Card
  - [ ] Accordion
  - [ ] Carousel (has caption background color options, can contain blockquotes)
- [x] Section (Light Gray)
  - [ ] Text (has background color options, can contain blockquotes)
  - [ ] Blockquote
  - [ ] Webform (has background color options)
  - [ ] Card
  - [ ] Accordion
  - [ ] Carousel (has caption background color options, can contain blockquotes)
- [ ] Section (Dark Gray)
  - [ ] Text (has background color options)
  - [ ] Blockquote
  - [ ] Webform (has background color options)
  - [ ] Card
  - [ ] Accordion
  - [ ] Carousel (has caption background color options, can contain blockquotes)
- [ ] Section (Black)
  - [ ] Text (has background color options, can contain blockquotes)
  - [ ] Blockquote
  - [ ] Webform (has background color options)
  - [ ] Card
  - [ ] Accordion
  - [ ] Carousel (has caption background color options, can contain blockquotes)
- [ ] Section (Gold)
  - [ ] Text (has background color options, can contain blockquotes)
  - [ ] Blockquote
  - [ ] Webform (has background color options)
  - [ ] Card
  - [ ] Accordion
  - [ ] Carousel (has caption background color options, can contain blockquotes)

**Comment by joewhitsitt** [comment]
Constructing a few pages on default dev for testing. Pull locally (docroot/sites/default/blt.yml), turn on uids_base and go to:

https://default.local.drupal.uiowa.edu/kitchen-sink
https://default.local.drupal.uiowa.edu/kitchen-sink-webform
https://default.local.drupal.uiowa.edu/kitchen-sink-carousel
